cognee/tasks/ingestion/ingest_data.py

```
@@ -5,21 +5,13 @@
 from cognee.modules.users.models import User
 from cognee.infrastructure.databases.relational import get_relational_config, get_relational_engine
 from cognee.modules.data.methods import create_dataset
+from cognee.shared.dlt_handler import dlt_destinations_sqlalchemy_handler
 from cognee.modules.users.permissions.methods import give_permission_on_document
 
 async def ingest_data(file_paths: list[str], dataset_name: str, user: User):
     relational_config = get_relational_config()
```

Ruff

Local variable `relational_config` is assigned to but never used

Remove assignment to unused variable `relational_config`

(F841)